tests/cpp/test_memory.cpp

```diff
@@ -2504,6 +2504,46 @@ TEST_P(LdMatrixTest, Regular) {
   testValidate(&fusion, cg_outputs, {t0}, __LINE__, __FILE__);
 }
 
+class StMatrixTest : public NVFuserTest{};
+
+TEST_F(StMatrixTest, Regular) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto operand = MmaOperand::B;
+
+  int sizeM = 8;
+  int sizeN = 8;
+
+  auto tv0 = makeContigConcreteTensor({sizeM, sizeN}, DataType::Half);
+  fusion.addInput(tv0);
+  // tv0 (global) -> tv1 (shared)
+  auto tv1 = set(tv0);
+  tv1->setMemoryType(MemoryType::Shared);
+  auto tv2 = set(tv1);
+  // tv1 (shared) -> tv2 (registers)
+  tv2->definition()->as<LoadStoreOp>()->setOpType(LoadStoreOpType::LdMatrix);
```

Comment:
Instead of using ldmatrix, could we use plain set op to load data directly from gmem to register? The reason for doing so is, I believe we should make unit test really "unit". For example, if there is a bug in the implementation of `ldmatrix`,  this test can still pass as long as long as it is wrong the same way as ldmatrix.

Reply:
Thanks! Please take another look at the test.